pkg/elements/webm.go

```diff
@@ -78,6 +100,9 @@ func (s *WebmSaver) pushOpus(sample *avp.Sample) {
 }
 
 func (s *WebmSaver) pushVP8(sample *avp.Sample) {
+	if !s.cfg.Video {
+		return
+	}
 	payload := sample.Payload.([]byte)
 	// Read VP8 header.
 	videoKeyframe := (payload[0]&0x1 == 0)
```

Comment:
@grahamking because of `if s.videoWriter == nil || s.audioWriter == nil` condition initWriter will be called for every key frame in video only file

Reply:
Thanks! I changed it to only check videoWriter.